test/sources/types.js

```diff
@@ -6,22 +6,22 @@ const Root = Protobuf.Root
 const Type = Protobuf.Type
 const Field = Protobuf.Field
```

Comment:
Nit / off-topic: It's probably possible to use above:

```javascript
import { Field, Root, Type } from 'protobufjs/light'
```

Also, the imports are inconsistent (`import` vs `require`).